segfault when deploying

reported in a support ticket.  I encouraged the reporter to follow this issue.

They are running on ubuntu 14.04 64 bit.

```netlifyctl -A ACCESS_TOKEN_REDACTED_BUT_WAS_VERIFIED_AS_VALID deploy

No site configured in the netlify.toml, fetching your existing sites.

Choose a site to deploy to or 0 to create a new site.

[0] Create a new site
[1] site1
[2] site2
[3] site3
[...]
[13] site13
=> Which site? (default: 0) 2

panic: runtime error: invalid memory address or nil pointer dereference

[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x7f98fd]

goroutine 1 [running]:

github.com/netlify/netlifyctl/operations.AskForInput(0x8f8869, 0xb, 0x8f18e7, 0x1, 0xc42015f5b0, 0x1, 0x1, 0x0, 0x0, 0x410760, ...)

/home/boxofmonocles/gocode/src/github.com/netlify/netlifyctl/operations/ui.go:57 +0x26d

github.com/netlify/netlifyctl/operations.ChooseOrCreateSite(0xc42012c180, 0x7f01a4f5a430, 0xc4202017a0, 0x7f01a4f5a430, 0xc4202017a0, 0x4118cc)

/home/boxofmonocles/gocode/src/github.com/netlify/netlifyctl/operations/site.go:86 +0x4a7

github.com/netlify/netlifyctl/commands/deploy.(*deployCmd).deploySite(0xc42041fe60, 0x7f01a4f5a470, 0xc4202017a0, 0xc420370b40, 0xc42012c100, 0x0, 0x2, 0x0, 0x0)

/home/boxofmonocles/gocode/src/github.com/netlify/netlifyctl/commands/deploy/deploy.go:48 +0x90e

github.com/netlify/netlifyctl/commands/deploy.(*deployCmd).(github.com/netlify/netlifyctl/commands/deploy.deploySite)-fm(0x7f01a4f5a470, 0xc4202017a0, 0xc420370b40, 0xc42012c100, 0x0, 0x2, 0x7f01a4f5a430, 0xc4204efa38)

/home/boxofmonocles/gocode/src/github.com/netlify/netlifyctl/commands/deploy/deploy.go:33 +0x69

github.com/netlify/netlifyctl/commands/middleware.DebugMiddleware.func1(0x7f01a4f5a470, 0xc4202017a0, 0xc420370b40, 0xc42012c100, 0x0, 0x2, 0x0, 0x0)

/home/boxofmonocles/gocode/src/github.com/netlify/netlifyctl/commands/middleware/middleware.go:67 +0x104

github.com/netlify/netlifyctl/commands/middleware.AuthMiddleware.func1(0x7f01a4f5a470, 0xc420201710, 0xc420370b40, 0xc42012c100, 0x0, 0x2, 0x1, 0xc420201680)

/home/boxofmonocles/gocode/src/github.com/netlify/netlifyctl/commands/middleware/middleware.go:86 +0x1d4

github.com/netlify/netlifyctl/commands/middleware.ClientMiddleware.func1(0x7f01a4f5a470, 0xc420201590, 0xc420370b40, 0xc42012c100, 0x0, 0x2, 0x88ebe0, 0x7f01a4f5a3b0)

/home/boxofmonocles/gocode/src/github.com/netlify/netlifyctl/commands/middleware/middleware.go:124 +0x26b

github.com/netlify/netlifyctl/commands/middleware.LoggingMiddleware.func1(0x7f01a4f5a470, 0xc420201590, 0xc420370b40, 0xc42012c100, 0x0, 0x2, 0x8f2c24, 0x4)

/home/boxofmonocles/gocode/src/github.com/netlify/netlifyctl/commands/middleware/middleware.go:75 +0x216

github.com/netlify/netlifyctl/commands/middleware.NewRunFunc.func1(0xc420370b40, 0xc42012c100, 0x0, 0x2, 0x0, 0x0)

/home/boxofmonocles/gocode/src/github.com/netlify/netlifyctl/commands/middleware/middleware.go:42 +0xf5

github.com/spf13/cobra.(*Command).execute(0xc420370b40, 0xc42041ffa0, 0x2, 0x2, 0xc420370b40, 0xc42041ffa0)

/home/boxofmonocles/gocode/src/github.com/spf13/cobra/command.go:643 +0x3f8

github.com/spf13/cobra.(*Command).ExecuteC(0xb4bda0, 0xb70c8b, 0x8f26e7, 0x3)

/home/boxofmonocles/gocode/src/github.com/spf13/cobra/command.go:733 +0x339

github.com/netlify/netlifyctl/commands.Execute()

/home/boxofmonocles/gocode/src/github.com/netlify/netlifyctl/commands/netlify.go:42 +0x2a5

main.main()

/home/boxofmonocles/gocode/src/github.com/netlify/netlifyctl/main.go:16 +0x7e
```

--- a/operations/site.go
+++ b/operations/site.go
@@ -83,7 +83,7 @@ func ChooseOrCreateSite(client *porcelain.Netlify, ctx context.Context) (*models
 	}
 
 	for {
-		input, err := AskForInput("Which site?", "0", nil)
+		input, err := AskForInput("Which site?", "0")
 		if err == nil {
 			if selection, ok := nameToId[input]; ok {
 				return selection, nil

